README.md

```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
 - Named as README (case-insensitive): `docs/Guides/README.mdx`
 - Same name as the parent folder: `docs/Guides/Guides.md`
 
-#### Referencing a page
+### Markdown Links and URLs
+
+The URL of a page will be composed of the file path to the page in context from the current file. All references to a another documentation page must end with the `.md` extension. Docusaurus will automatically remove the `.md` extension from the URL. The file path is needed for Docuasurus to generate the correct URL for the page when versioning is enabled.
 
-The url of a page will be composed from the path of the file relative to the `content` directory.
 
-**Example** docs/content/1-introduction/1-what-is.md will have http://localhost:9000/introduction/what-is as the url
 
-In markdown you can reference this page relatively to the root of the domain using this syntax:
+The following example shows how to reference a page in various scenarios. Assume you have the following folder structure when reviewing the examples below:
+
+```shell
+.
+└── docs
+    └── docs-content
+        ├── architecture
+        │   ├── grpc.md
+        │   └── ip-addresses.md
+        ├── aws
+        │   └── iam-permissions.md
+        ├── clusters
+        └── security.md
+```
+
+#### Same Folder
+
```

Comment:
It's up to you - you could say "To ..." instead of "If you want to..." 
What you have here isn't incorrect, so it's your choice.